src/v2/Components/v2/ArtworkFilter/ArtworkFilters/__tests__/PriceRangeFilter.jest.tsx

```diff
@@ -86,32 +86,52 @@ describe("PriceRangeFilter", () => {
     expect(wrapper.find(Input).last().html()).toContain('value="1000"')
   })
 
-  // TODO: Unable to trigger this changes correctly
```

Comment:
https://github.com/enzymejs/enzyme/issues/1943

I don't know how this is a real issue in this library since it just seems like something that should definitely work... so guess I just won't use `simulate` anymore.

Reply:
/shrug

I haven't encountered _too_ much weirdness in using enzyme + simulate, but I believe it. Front-end testing frameworks are like politicians...